lib/patternfly/dist/css/patternfly-additions.css

```diff
@@ -2885,7 +2885,8 @@ table.datatable th:active {
 }
 @font-face {
   font-family: 'PatternFlyIcons-webfont';
-  src: url('fonts/patternfly.woff') format('woff');
+  src: url('../fonts/PatternFlyIcons-webfont.eot');
+  src: url('../fonts/PatternFlyIcons-webfont.eot?#iefix') format('embedded-opentype'), url('../fonts/PatternFlyIcons-webfont.ttf') format('truetype'), url('../fonts/PatternFlyIcons-webfont.woff') format('woff'), url('../fonts/PatternFlyIcons-webfont.svg#PatternFlyIcons-webfont') format('svg');
```

Comment:
I've verified that the patternfly CSS installed and distributed by Cockpit does not have this multi url line. It merely shows up here as the original file, which is amended during the build with our sed script.